Modules/ITS/include/ITS/ITSDecodingErrorCheck.h

```diff
@@ -40,6 +40,34 @@ class ITSDecodingErrorCheck : public o2::quality_control::checker::CheckInterfac
   // Override interface
   Quality check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap) override;
   void beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult = Quality::Null) override;
+  std::vector<int> vDecErrorLimits, vListErrorIdBad, vListErrorIdMedium;
+  bool doFlatCheck = false;
+
+  TString sErrorDesc[23] = {
```

Comment:
instead of defining the local list of names you can do:
```
o2::itsmft::GBTLinkDecodingStat statistics;
statistics.ErrNames[<some_number>]; //to get the string you have defined locally
```
In this way if there is an update in O2, we have it for free also in our QC

Reply:
fixed